docs/source/testing/conf.py

```diff
@@ -0,0 +1,239 @@
+    # -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/stable/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+# import recommonmark
+# from recommonmark.parser import CommonMarkParser
+# from recommonmark.transform import AutoStructify
+import m2r # Used to convert .md files into .rst files when Sphinx is building docs
```

Comment:
Should this `import m2r` line be removed?

Reply:
Resolved